python/iceberg/hive/hive_tables.py

```diff
@@ -34,12 +39,40 @@ def __init__(self, conf):
     def new_table_ops(self, conf, database, table):
         return HiveTableOperations(conf, self.get_client(), database, table)
 
-    def drop(self, database, table):
-        raise RuntimeError("Not yet implemented")
+    def drop(self, database, table, purge=False) -> None:
```

Comment:
can you add mypy annotations to the arguments here

Reply:
Done